### ISSUE TITLE
Were you able to figure this out?

### ISSUE THREAD

**Issue by sfc-gh-pkommini** [trysten/tree-sitter-sql#1]
Were you able to figure this out?

**Comment by trysten** [comment]
No. I got stuck on a [complex bug](https://github.com/tree-sitter/tree-sitter/issues/238). I still want to develop this grammar, but I couldn't proceed without understanding that darn bug! I'll give it a poke, thanks for the reminder.
